processing/src/main/java/io/druid/segment/incremental/IncrementalIndexAdapter.java

```diff
@@ -269,32 +266,15 @@ public ColumnCapabilities getCapabilities(String column)
   }
 
   @Override
-  public BitmapIndexSeeker getBitmapIndexSeeker(String dimension)
+  public IndexedInts getBitmapIndex(String dimension, int dictId)
   {
-    final Map<String, MutableBitmap> dimInverted = invertedIndexes.get(dimension);
-    if (dimInverted == null) {
-      return new EmptyBitmapIndexSeeker();
+    if (dictId >= 0) {
+      final Indexed<String> dimValues = getDimValueLookup(dimension);
+      String value = Strings.nullToEmpty(dimValues.get(dictId));
```

Comment:
is there a reason to call nullToEmpty here? this seems it might be an artifact of wrapping DimDim with NullValueConverterDimDim, however getBitmapIndex relies on the actual values stored in DimDim, not the values returned by the wrapper, it that correct?